disnake/components.py

```diff
@@ -110,7 +114,7 @@ class ActionRow(Component):
     ------------
     type: :class:`ComponentType`
         The type of component.
-    children: List[:class:`Component`]
+    children: List[Union[:class:`Button`, :class:`SelectMenu`, :class:`TextInput`]]
```

Comment:
TextInputs are not supported in ActionRows.

Reply:
Yes they are

Reply:
discussed on discord, didn't update here, sorry 😅 


I did not read the documentation properly.